src/pages/ReimbursementAccount/BeneficialOwnerInfo/substeps/CompanyOwnersListUBO.tsx

```diff
@@ -70,7 +70,7 @@ function CompanyOwnersListUBO({
 
     const isLoading = reimbursementAccount?.isLoading ?? false;
     const requestorData = getSubstepValues(REQUESTOR_PERSONAL_INFO_KEYS, {}, reimbursementAccount);
-    const error = ErrorUtils.getLatestErrorMessage(reimbursementAccount);
+    const error = reimbursementAccount ? ErrorUtils.getLatestErrorMessage(reimbursementAccount) : '';
```

Comment:
I think `getLatestErrorMessage` is performing this check inside

Reply:
noppe, but I've changed that